:py:mod:`spark_auto_mapper_fhir.value_sets.charge_item_status`
==============================================================

.. py:module:: spark_auto_mapper_fhir.value_sets.charge_item_status


Module Contents
---------------

Classes
~~~~~~~

.. autoapisummary::

   spark_auto_mapper_fhir.value_sets.charge_item_status.ChargeItemStatusCode
   spark_auto_mapper_fhir.value_sets.charge_item_status.ChargeItemStatusCodeValues




.. py:class:: ChargeItemStatusCode(value)

   Bases: :py:obj:`spark_auto_mapper_fhir.value_sets.generic_type.GenericTypeCode`

   ChargeItemStatus
   From: http://hl7.org/fhir/chargeitem-status in valuesets.xml
       Codes identifying the lifecycle stage of a ChargeItem.

   .. py:attribute:: codeset
      :annotation: :spark_auto_mapper_fhir.fhir_types.uri.FhirUri = http://hl7.org/fhir/chargeitem-status

      


.. py:class:: ChargeItemStatusCodeValues

   The charge item has been entered, but the charged service is not  yet
   complete, so it shall not be billed yet but might be used in the context of
   pre-authorization.
   From: http://hl7.org/fhir/chargeitem-status in valuesets.xml

   .. py:attribute:: Planned
      

      The charge item is ready for billing.
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: Billable
      

      The charge item has been determined to be not billable (e.g. due to rules
      associated with the billing code).
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: NotBillable
      

      The processing of the charge was aborted.
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: Aborted
      

      The charge item has been billed (e.g. a billing engine has generated financial
      transactions by applying the associated ruled for the charge item to the
      context of the Encounter, and placed them into Claims/Invoices.
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: Billed
      

      The charge item has been entered in error and should not be processed for
      billing.
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: EnteredInError
      

      The authoring system does not know which of the status values currently
      applies for this charge item  Note: This concept is not to be used for "other"
      - one of the listed statuses is presumed to apply, it's just not known which
      one.
      From: http://hl7.org/fhir/chargeitem-status in valuesets.xml


   .. py:attribute:: Unknown
      

      


